savemphtxt.m

```diff
@@ -40,36 +38,38 @@ function savemphtxt(node, face, elem, filename)
 fprintf(fp,'0 0 1\n4 Mesh\n2\n3\n%d\n1\n',n_node);
 
 % Write Node information
-for i = 1:n_node
-    fprintf(fp,'%.16f %.16f %.16f\n',node(i,1),node(i,2),node(i,3));
-end
+
+fprintf(fp,'%.16f %.16f %.16f\n',node(:,1:3).');
+
 fprintf(fp, '\n2\n\n3 tri\n');
 
 % Write Tri information
 fprintf(fp, '\n3\n');
 fprintf(fp, '%d\n\n',n_face);
-for i = 1:n_face
-    fprintf(fp,'%d %d %d\n',face(i,1),face(i,2),face(i,3));
-end
+% for i = 1:n_face

```

Comment:
please remove these commented lines if they are no longer needed. git keeps track of all edit history, so no need to keep those in the code.